bandit/bandit_report.js

```diff
@@ -3,16 +3,36 @@
 
 const { config } = require('../config')
 const { getAnnotation } = require('./bandit_annotations')
+const { countIssueLevels } = require('../annotations_levels')
 
-function customSummary (banditSummary) {
-  let severityHigh = banditSummary['SEVERITY.HIGH']
-  let severityMedium = banditSummary['SEVERITY.MEDIUM']
-  let severityLow = banditSummary['SEVERITY.LOW']
+/**
+ * @param {*} issues the issues found by Bandit
+ */
+function createMoreInfoLinks (issues) {
+  let issuesMap = new Map()
+  let moreInfo = 'For more information about the issues follow the links: \n'
+
+  for (let issue of issues) {
+    if (issuesMap.has(issue.test_id) === false) {
+      issuesMap.set(issue.test_id)
+      const text = `${issue.test_id}:${issue.test_name}`
+      moreInfo += `[${text}](${issue.more_info})\n`
+    }
+  }
+  return moreInfo
+}
+
+/**
+ * @param {*} annotations all issues found by Bandit wrapped in the annotation object
+ * see: https://developer.github.com/v3/checks/runs/#annotations-object
+ */
+function customSummary (annotations) {
```

Comment:
This function is an extra layer of indirection that does nothing: it extracts the returned values from `countIssueLevels` and rebinds them to the same names.
You can drop it and call `countIssueLevels` directly.

Reply:
Yep, you are right.